scripts/assert-free-ports.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 
-lsof -i :3000 > /dev/null && echo 'Refusing to run because port 3000 is already in use' && exit 0
-lsof -i :3883 > /dev/null && echo 'Refusing to run because port 3883 is already in use' && exit 0
\ No newline at end of file
+lsof -i :3000 > /dev/null && echo 'Refusing to run because port 3000 is already in use' && exit 1
+lsof -i :3883 > /dev/null && echo 'Refusing to run because port 3883 is already in use' && exit 1
+
+exit 0
```

Comment:
without this, the exit code is 1 when neither lines 3 nor 4 obtain.  I am not great at bash so I couldn't tell you why, but you can verify by running:
`./scripts/assert-free-ports.sh; echo $?`

If you have web or cli running you should see:
```
Refusing to run because port 3883 is already in use
1
```

Otherwise you should see:
```
./scripts/assert-free-ports.sh; echo $?
0
```